.pre-commit-config.yaml

```diff
@@ -17,6 +17,14 @@ repos:
         name: pytest
         entry: poetry run bash scripts/test-coverage-xml.sh
         language: system
-        files: ^src/zenml/|^tests/
+        files: ^src/zenml/|^tests/|^examples/
```

Comment:
I think this is not what we actually want here:
The linting script only runs for files inside the `src/zenml` and `tests` directory (and it will never work with mypy for all our current examples), so adding the examples directory here basically runs the linting script on all our src/test files even if you only modify an example file.

Reply:
(This should actually be a comment to the linting and not the testing script)